scripts/Auto-install-macos.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/zsh
+# A script to autoamtically install jellyfin-rpc on mac
+clear
+
+# Get the user's server information
+vared -p "Jellyfin Server URL (include http/https): " -c jellyfinurl
+vared -p "Jellyfin API Key (you can find this at ${jellyfinurl}/web/#!/apikeys.html): " -c jellyfinkey
+vared -p "Jellyfin Username: " -c jellyfinuser
+
+# Prompt the user for what libraries should be included/blocked
+responses=()
+
+vared -p "Include Movies in Discord Rich Presence? (y/n) " -c moviesEnabled
+if [[ $moviesEnabled == [Nn]* ]]; then
+  responses+=( "movie" )
+fi
+
+vared -p "Include TV in Discord Rich Presence? (y/n) " -c tvEnabled
+if [[ $tvEnabled == [Nn]* ]]; then
+  responses+=( "episode" )
+fi
+
+vared -p "Include Music in Discord Rich Presence? (y/n) " -c musicEnabled
+if [[ $musicEnabled == [Nn]* ]]; then
+  responses+=( "music" )
+fi
+
+vared -p "Include Live TV in Discord Rich Presence? (y/n) " -c livetvEnabled
+if [[ $livetvEnabled == [Nn]* ]]; then
+  responses+=( "livetv" )
+fi
+
+# Build the blocklist string
+if [[ ${#responses} -eq 0 ]]; then
+  blocklistString='["movie", "episode", "music", "livetv"]'
+else
+  blocklistString='['
+  for i in ${responses[@]}; do
+    blocklistString+="\"$i\", "
+  done
+  blocklistString=${blocklistString%?}
+  blocklistString=${blocklistString%?}
+  blocklistString+=']'
+fi
+
+#get discord application ID, or use default if left blank
+vared -p "Discord Application ID (leave blank if you're unsure): " -c discordAppId
+if [ -z $discordAppId ]; then
+  discordAppId="1053747938519679018"
+fi
+
+# Get Imgur client ID & enable/disable image uploading
+configImgurEnabled="false"
+vared -p "Upload album artwork to Imgur to be displayed in discord? (y/n) " -c artworkEnabled
+if [[ $artworkEnabled == [Yy]* ]]; then
+  configImgurEnabled="true"
+  echo "To get an Imgur client ID, go to https://api.imgur.com/oauth2/addclient"
+  echo "Name can be anything, authorization type must be 'OAuth 2 authorization without a callback URL'"
+  echo "Press submit to get your client ID"
+  echo "NOTE: Port formwarding must be enabled on your server to send images"
+  vared -p "Imgur client ID (): " -c imgurId
+fi
```

Comment:
Enable images can be toggled without imgur if its port forwarded, after saying no to this it should prompt to enable images without imgur too

Reply:
Port forwarding isnt required when using imgur, I just noticed it says that

Reply:
Apologies for my confusion, I believe the information and instruction in [45582b4](https://github.com/Radiicall/jellyfin-rpc/pull/18/commits/45582b4ffe76510eb87e054575cba9818883b926#diff-3807b76448243f6b4d1b13c1f1b0f55ac4a802675378d7fd3fc76ba6e9a2e64cR74-R91) is now correct.

Now when a user enables images, they are given the option to enable Imgur integration, and it is explained that Imgur is required if the server is not port-forwarded.